pkg/cli/mirror/mirror.go

```diff
@@ -103,7 +103,7 @@ func (o *MirrorOptions) Complete(args []string) error {
 	return nil
 }
 
-func (o *MirrorOptions) Validate() error {
+func (o *MirrorOptions) Validate(args []string) error {
```

Comment:
Can you add a unit test for this

Reply:
Will do

Reply:
Also #224 has some changes that improve validation in `Complete()` relevant here.